node/chainSimulator/chainSimulator.go

```diff
@@ -195,6 +207,41 @@ func (s *simulator) GetInitialWalletKeys() *dtos.InitialWalletKeys {
 	return s.initialWalletKeys
 }
 
+// AddValidatorKeys will add the provided validators private keys in the keys handler on all nodes
+func (s *simulator) AddValidatorKeys(validatorsPrivateKeys *dtos.ValidatorsKeys) error {
+	s.mutex.Lock()
+	defer s.mutex.Unlock()
+
+	for shard, node := range s.nodes {
+		for idx, privateKeyHex := range validatorsPrivateKeys.PrivateKeysBase64 {
+			decodedPrivateKey, err := base64.StdEncoding.DecodeString(privateKeyHex)
```

Comment:
unnecessary complications. We do not need the `dtos.ValidatorsKeys` struct.
We can provide the private keys as a [][]byte, directly.

Reply:
done